### ISSUE TITLE
[BUG] Sphere-sphere collision response

### ISSUE THREAD

**Issue by emilk** [dimforge/rapier#96]
Here are two balls, `radius=0.5` at `[0.0, 0.5, 0.0]` and `[0.1, 1.5, 0.0]` on a floor at `y=0`. `friction=1`, `restitution=0` (balls and floor). Everything else default.

![Kapture 2021-01-25 at 13 35 31](https://user-images.githubusercontent.com/1148717/105706738-377e3480-5f12-11eb-99f5-c8afaef619df.gif)

The red dots are the reported contact points, drawn like this:

``` rust
for contact_pair in self.narrow_phase.contact_graph().interactions() {
    for manifold in &contact_pair.manifolds {
        if let Some(collider1) = self.colliders.get(manifold.pair.collider1) {
            if let Some(collider2) = self.colliders.get(manifold.pair.collider2) {
                for contact in manifold.active_contacts() {
                    let world_p1 = collider1.position() * contact.local_p1;
                    let world_p2 = collider2.position() * contact.local_p2;
                    draw_red_sphere(world_p1);
                    draw_red_sphere(world_p2);
                }
            }
        }
    }
}
```

It looks like the contact is computed with the wrong rotation somewhere (it seems to rotate in the opposite direction of the top sphere).

**Comment by sebcrozet** [comment]
Thank you for the test case. This has been observed in 2D too: https://github.com/dimforge/rapier/issues/76
This has been fixed in #79 (not merged/released yet).

**Comment by emilk** [comment]
Nice! Looking forward to https://github.com/dimforge/rapier/pull/79. Feel free to close this as a duplicate of #76, or keep it as something to test once https://github.com/dimforge/rapier/pull/79 is merged.
